### DISCUSSION TITLE
Renovate not able to find argocd chart dependency

### DISCUSSION THREAD

**Discussion by rajat053117** [renovatebot/renovate#29243] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

Team , I am using Renovate and i have a argocd bootstrap like this but it is not able to find the dependencies 
Please help where i am wrong

```

export ARGOCD_CHART_VERSION=5.46.7 # renovate: depName=argoproj/argo-helm extractVersion=^argo-cd-(?<version>.+)$

```

and a regex like this 

```
    {
      customType: 'regex',
      fileMatch: [
        'scripts/argocd-bootstrap.sh',
      ],
      matchStrings: [
        'export \w+_VERSION=(?<currentValue>[0-9.]+)\s+# renovate: depName=(?<depName>[^ \\n]+) +(extractVersion=(?<extractVersion>[^\\n]+))?',
      ],
      datasourceTemplate: 'github-releases',
    },
    
    ```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by secustor** [comment]
Do you have tested your regex with regex101? 
https://docs.renovatebot.com/string-pattern-matching/#use-regex101-to-test-your-patterns

**Reply by rajat053117** [reply]
Yeah I have tested it and it is able to match

**Reply by secustor** [reply]
Regexes are not evaluated recursively, you can only provide plain strings in the file content.

**Reply by rajat053117** [reply]
This is what i am using https://regex101.com/r/9cxYkk/1 
Please let me know where i am going wrong
